### PR TITLE
Phpdev async with amphp

### PR DESCRIPTION
**Pull request by unglaublicherdude**
So basically you can now require a specific branch.

Branches that are named like phpdev-* are pushed without the phpdev- prefix to the vaas-php repository.

So when you have a branch called phpdev-somethinggreat you can install the branch with
```bash
composer require gdata/vaas:dev-somethinggreat
```

### CODE REVIEW

**Comment by unglaublicherdude** [comment]
This PR also replaces this https://github.com/GDATASoftwareAG/vaas/pull/544

**Comment by unglaublicherdude** [comment]
You can see the asnyc version running in this https://github.com/GDATASoftwareAG/nextcloud-gdata-antivirus/pull/117

**Comment by unglaublicherdude** [comment]
This has to be a major release because there is a big api change

File: php/examples/VaasExample/GetVerdictByFile.php [hunk 1]
```diff
@@ -13,9 +13,9 @@
...
 );
 
-$vaas = new Vaas(
-    getenv("VAAS_URL") ?? "wss://gateway.production.vaas.gdatasecurity.de"
-);
+$vaas = (new Vaas())
+    ->WithUrl(getenv("VAAS_URL") ?? "wss://gateway.production.vaas.gdatasecurity.de")
+    ->build();
 
 $vaas->Connect($authenticator->getToken());
```
Comment (unglaublicherdude) [inline php/examples/VaasExample/GetVerdictByFile.php:20]: Just to be clear. With the new implementation the Connect is not necessary anymore. Maybe I sould also give an example on how to use it without you having to call connect yourself.

Comment (unglaublicherdude) [reply php/examples/VaasExample/GetVerdictByFile.php]: If fixed that in my new commit.

File: php/src/vaas/VaasConnection.php
Comment (pstadermann) [inline php/src/vaas/VaasConnection.php:1]: Functions should camelCased

File: php/src/vaas/Vaas.php
Comment (pstadermann) [inline php/src/vaas/Vaas.php:1]: Methods should be camelCase. E.g. the builder uses .With*(), but .build().

**Review by pstadermann — APPROVED** [review]

**Pull request merged**